### PR TITLE
[DOCS] Update to cluster docs

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
This PR single-sources a few pieces of content around the cluster quoickstarts, and adds a bit more explanation around things. It also fixes a few small steps for the K8s guide.

The K8s and binaries cluster guides are a little different from each other right now, mostly as I have had a lot of issues getting a local K8s cluster to work properly, and there are so many ways people have this setup anyway… Over time I would like to improve it…

### CODE REVIEW

File: site/content/includes/m3query/annotated_config.yaml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+  serviceName: <string>
+  # Tracing backen to use, valid options: [jaeger, lightstep]
+  backend: <string>
+  # If using Jaeger, options to send to tracing backend
+  jaeger:
+  # If using Lightstep, options to send to tracing backend
+  lightstep:
+
+clusters:
+  - namespaces:
```
Comment (nbroyles) [inline site/content/includes/m3query/annotated_config.yaml:52]: The namespaces section can be removed here.

File: site/content/includes/m3query/annotated_config.yaml [hunk 2]
```diff
@@ -0,0 +1,246 @@
+# The server listen address
```
Comment (nbroyles) [inline site/content/includes/m3query/annotated_config.yaml:1]: Generally, I think this config needs to be updated. I'm not sure it ever got changed after 1.0. Maybe not something we need to do for this PR, but worth us doing

**Review by nbroyles — APPROVED** [review]

File: site/content/includes/m3query/annotated_config.yaml (hunk 2 above)
Comment (ChrisChinchilla) [reply site/content/includes/m3query/annotated_config.yaml:1]: @nbroyles I have that in progress right now, and boy is it a lot of work!

File: site/content/includes/m3query/annotated_config.yaml (hunk 1 above)
Comment (ChrisChinchilla) [reply site/content/includes/m3query/annotated_config.yaml:52]: I'll save that for the other in progress PR, but thanks for the note :)

**Pull request merged**